### PR TITLE
Add cpu config

### PR DESCRIPTION
**Pull request by yannjor**
Adds cpu config with option to display core count. Also removes some
unnecessary words from the cpu model name.

### CODE REVIEW

File: src/cpu.rs [hunk 1]
```diff
@@ -27,31 +37,48 @@ fn parse(contents: &str) -> HashMap<&str, &str> {
...
-    Some(cpu_info.get("model name")?.to_string())
+    let model = cpu_info
+        .get("model name")
+        .expect("Failed to parse CPU model")
+        // Remove some unnecessary patterns from model name
+        .replace("(TM)", "")
+        .replace("(R)", "")
+        .replace(" Core", "")
+        .replace(" CPU", "")
+        .replace(" Processor", "")
```
Comment (isacsund) [inline src/cpu.rs]: hmm... what does the final model look like for you?

Comment (isacsund) [reply src/cpu.rs]: this is what i get: `CPU: AMD Ryzen 5 2600 Six-Core (6)`

Comment (yannjor) [reply src/cpu.rs]: > hmm... what does the final model look like for you?

Now looks like this: `Intel i5-8400 @ 2.80GHz`. Previously looked like this: `Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz`

Comment (yannjor) [reply src/cpu.rs]: How does it look like for you?

Comment (yannjor) [reply src/cpu.rs]: > this is what i get: `CPU: AMD Ryzen 5 2600 Six-Core (6)`

Hmm, maybe that 'Six-Core' part is a bit unnecessary. Neofetch has a long [list](https://github.com/dylanaraps/neofetch/blob/master/neofetch#L2345-#L2366) of patterns that they remove from the model name....

Comment (isacsund) [reply src/cpu.rs]: maybe we can have the same

Comment (yannjor) [reply src/cpu.rs]: Then we would probably have to use the regex library, which would add another dependency but maybe that's ok 😄

Comment (isacsund) [reply src/cpu.rs]: i was thinking more of having a list with patterns defined and then just loop through them and replace with empty strings (for now)

**Pull request merged**